proxy/reverse.go

```diff
@@ -98,6 +102,8 @@ func (p *reverseProxy) ServeHTTP(rw http.ResponseWriter, clientreq *http.Request
 			case <-closeNotifier.CloseNotify():
 				tp, ok := p.transport.(*http.Transport)
 				if ok {
+					atomic.StoreInt32(&requestCanceled, 1)
+					log.Printf("proxy: request from %v canceled", clientreq.RemoteAddr)
```

Comment:
I think it helps you to tune the proxy timeout values. (e.g. my other PR sets the dial timeout to 1 second, which seems not optimal with the default timeout set in fleet which is also 1 second)